app/helpers/application_helper.rb

```diff
@@ -30,8 +30,7 @@ def session_with_trust?
   end
 
   def page_with_trust?
-    current_page?(controller: 'sign_up/passwords', action: 'new') ||
-      current_page?(controller: 'users/reset_passwords', action: 'edit')
+    !current_page?(controller: 'users/sessions', action: 'new')
```

Comment:
this is making sure we don't do google analytics on pages that prompt for passwords, right?, we should still exclude the reset passwords page?

Reply:
the goal is to only load DAP on the login page, and anything else is considered a page with trust

this could be clearer, maybe something like:

```
return false if current_page?(controller: 'users/sessions', action: 'new')
return true
```
?

Reply:
oh in that case, maybe we just invert the entire method?

```ruby
def load_dap?
  current_page?(controller: 'users/sessions', action: 'new')
end
```

and then flip the call site too

Reply:
dang, just realized it's also used elsewhere:
https://github.com/18F/identity-idp/blob/master/app/views/layouts/base.html.erb#L16-L18

I went with my first comment in 7b960c92b but am very open to other ideas